guides/customization/tailscale.md

```diff
@@ -78,48 +84,37 @@ the target files, in a subdirectory called `files`:
 ```
 
 While it is possible to configure everything directly in a single Dockerfile, we
-recommend using a folder hierarchy, since it makes it easier to create a
+recommend using a folder hierarchy, since this makes it easier to create a
 reproducible image and examine the change history for individual files.
 
-Create the folder hierarchy using:
+Create the folder hierarchy:
 
 ```console
-$ mkdir --parents --verbose \
+mkdir --parents --verbose \
     files/etc/apt/preferences.d \
     files/etc/apt/sources.list.d \
     files/etc/systemd/system/tailscaled.service.d \
     files/usr/share/keyrings
-mkdir: created directory 'files'
-mkdir: created directory 'files/etc'
-mkdir: created directory 'files/etc/apt'
-mkdir: created directory 'files/etc/apt/preferences.d'
-mkdir: created directory 'files/etc/apt/sources.list.d'
-mkdir: created directory 'files/etc/systemd'
-mkdir: created directory 'files/etc/systemd/system'
-mkdir: created directory 'files/etc/systemd/system/tailscaled.service.d'
-mkdir: created directory 'files/usr'
-mkdir: created directory 'files/usr/share'
-mkdir: created directory 'files/usr/share/keyrings'
 ```
 
-## Step 1a: Add image repository
+### Step 1a: Add image repository
 
 Add the Tailscale package repository to `tailscale.list` in the local path
-`files/etc/apt/sources.list.d`, which will appear in `/etc/apt/sources.list.d`
-in the resulting image, with the following contents:
+`files/etc/apt/sources.list.d`. This will appear in `/etc/apt/sources.list.d` in
+the resulting image, with the following contents:
 
 ```text
-# Tailscale packages for ubuntu focal
+# Tailscale packages for Ubuntu Focal Fossa
```

Comment:
the comment came from the Tailscale download instructions, we could remove the comment entirely

https://tailscale.com/download